triage/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8" />
+<link rel="stylesheet" type="text/css" href="style.css">
+<title>Kubernetes Aggregated Test Results</title>
+</head>
+<body>
+<h1>Kubernetes Failures</h1>
+<form id="options" onchange="rerender();">
+Sort by
+<label><input id="sort" name="sort" type="radio" value="total">total count</label>
+<label><input name="sort" type="radio" value="day" checked>count in last day</label>
+<label><input name="sort" type="radio" value="message">error message</label>
+<br>
+<label style="display:none"><input type="checkbox" id="show-normalize">Display normalized errors</label><br>
+Include results from:
+<label><input type="checkbox" checked id="job-ci">CI</label>
+<label><input type="checkbox" id="job-pr">PR</label>
+<br><br>
+Filter (these are regexes):
+<table>
+<label><tr><td>Failure text<td><input type="text" id="filter-text"></label><br>
+<label><tr><td>Job<td><input type="text" id="filter-job"></label><br>
+<label><tr><td>Test<td><input type="text" id="filter-test"></label><br>
+</table>
+</form>
+<div id="summary"></div>
+<div id="clusters">
+  <h2>Loading... <span id="loading-progress"></span></h2>
```

Comment:
@rmmh @michelle192837 this is a really nice pattern. Can we add this to testgrid some time this quarter?